daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -69,6 +69,10 @@ object LanguageVersion {
   val EarlyAccessVersions: VersionRange[LanguageVersion] =
     StableVersions
 
+  // Only the current stable version, no older versions.
+  val StableOnlyVersions: VersionRange[LanguageVersion] =
```

Comment:
```suggestion
  val LastestOnly: VersionRange[LanguageVersion] =
```

Reply:
In engine world Stable means non developments.

Reply:
Isn't that more confusing, with the distinction between stable and latest version?

Reply:
More confusing, for sure no ;). 
You maybe right, that it is maybe not less confusing. 
What about  `LatestStableVersion` ?

Reply:
I mean, ... what does a "latest stable version" mean when we have "latest version" and a "stable version" and these aren't always the same thing? 🤔

Reply:
The word "stable" in the code of the engine has a different meaning that in the world "stable" in the compiler code  
- "stable" in the engine means not development or nor preview version (there is no stable version, there are stable versions)
- "stable" in the compiler mean default output version. (there is only one stable version)
 
In this precise context there it does not make sens to add `OnlyStableVersions` because `StableVersions` already contains only stable versions. 

I am open to clean up this inconsistency but in a different PR (potentially merged before).